### ISSUE TITLE
Error parsing 'banner' for Cisco IOS (with extra characters after the banner delimiter)

### ISSUE THREAD

**Issue by mynameisosama** [mpenning/ciscoconfparse#111]
I had to change the BANNER_STR_RE to make the _banner_mark_regex function work.
command:
```
banner motd ^CC
************************************************************
*  WARNING! WARNING! WARNING! WARNING! WARNING! WARNING!   *
************************************************************
* Access to and use of this device and/or other devices is *
* restricted to authorized users only. Unauthorized        *
* individuals attempting to access this device may be      *
* subject to prosecution.                                  *
************************************************************
^C
```

change:
original: BANNER_STR_RE = r'^(?:(?P<btype>(?:set\s+)*banner\s\w+\s+)(?P<bchar>\S)(?:\S)?)$'
new: BANNER_STR_RE = r'^(?:(?P<btype>(?:set\s+)*banner\s\w+\s+)(?P<bchar>\S)(?:\S+)?)$'

**Comment by mpenning** [comment]
I appreciate your pull request; however, I prefer to correct a problem that still remained in it.

In the fix I'm using this:
```python
BANNER_STR_RE = r'^(?:(?P<btype>(?:set\s+)*banner\s\w+\s+)(?P<bchar>\S))'
```

If we use the PR, there are two issues:
1. The PR only fixed ```IOSConfigList()```, and did not fix ```NXOSConfigList()```, which had the same problem.
2.  I would rather simplify ```BANNER_STR_RE``` to remove parsing all the way to the end of the line.

**Comment by mpenning** [comment]
This issue is fixed in version 1.3.19
